examples/hello-world/hello-flower/README.md

```diff
@@ -49,3 +52,22 @@ the TensorBoard metrics to the server at each iteration using NVFlare's metric s
 ```bash
 python job.py --job_name "flwr-pt-tb" --content_dir "./flwr-pt-tb" --stream_metrics
 ```
+
+You can visualize the metrics streamed to the server using TensorBoard.
+```bash
+tensorboard --logdir /tmp/nvflare/hello-flower
+```
+![tensorboard training curve](./train.png)
+
+## Notes
+Make sure your `pyproject.toml` files in the Flower apps contain an "address" field. This needs to be present as the `--federation-config` option of the `flwr run` command tries to override the `“address”` field.
+Your `pyproject.toml` should include a section similar to this
+```
+[tool.flwr.federations]
+default = "xxx"
+
+[tool.flwr.federations.xxx]
+options.num-supernodes = 2
```

Comment:
Should mention that number of supernodes must equal to the number of FL sites.

Reply:
Added this to notes in readme

"The number `options.num-supernodes` should match the number of NVFlare clients defined in [job.py](./job.py), e.g., `job.simulator_run(args.workdir, gpu="0", n_clients=2)`."